src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DscExtensionTests.ps1

```diff
@@ -103,10 +103,10 @@ function Test-GetAzureRmVMDscExtension
     finally
     {
 		# Cleanup
-		if(Get-AzureRmResourceGroup -Name $rgname -Location $loc)
-		{
+		#if(Get-AzureRmResourceGroup -Name $rgname -Location $loc)
```

Comment:
Commented code should be removed. Since the `finally` has no code, it should be removed too.